packages/dds/tree/src/tree/treeTextFormat.ts

```diff
@@ -168,14 +169,33 @@ export function genericTreeKeys<T>(tree: GenericFieldsNode<T>): readonly FieldKe
         if (global === undefined) {
             return [];
         }
-        return (Object.getOwnPropertyNames(global) as GlobalFieldKey[]).map(symbolFromKey);
+        return (Object.getOwnPropertyNames(global) as GlobalFieldKey[])
+            .filter((key: GlobalFieldKey) => {
+                const value: Value = global[key];
```

Comment:
this is a field. Do not use the type "Value" for jsonableTree fields. Those have type JsonableTree[]